packages/runtime/test/src/interpreter/opcode-list.ts

```diff
@@ -7184,4 +7185,72 @@ describe("Teal Opcodes", function () {
 		});
 	});
 
+	describe("Ed25519verify_bare", function () {
+		const stack = new Stack<StackElem>();
+
+		it("should push 1 to stack if signature is valid", function () {
+			const account = generateAccount();
+			const toSign = new Uint8Array(Buffer.from([1, 9, 25, 49]));
+			const signed = signBytes(toSign, account.sk);
+
+			stack.push(toSign); // data
+			stack.push(signed); // signature
+			stack.push(decodeAddress(account.addr).publicKey); // pk
+
+			const op = new Ed25519verify_bare([], 1);
+			op.execute(stack);
+			const top = stack.pop();
+			assert.equal(top, 1n);
+		});
+
+		it("should push 0 to stack if signature is invalid", function () {
+			const account = generateAccount();
+			const toSign = new Uint8Array(Buffer.from([1, 9, 25, 49]));
+			const signed = signBytes(toSign, account.sk);
+			signed[0] = (Number(signed[0]) + 1) % 256;
+
+			stack.push(toSign); // data
+			stack.push(signed); // signature
+			stack.push(decodeAddress(account.addr).publicKey); // pk
+
+			const op = new Ed25519verify_bare([], 1);
+			op.execute(stack);
+			const top = stack.pop();
+			assert.equal(top, 0n);
+		});
+
+		it(
+			"should throw invalid type error Ed25519verify",
```

Comment:
```suggestion
			"should throw an invalid type error",
```

Reply:
There is no need for repetition of the `Ed25519verify` if we already have the `describe` section